src/Dropdown.js

```diff
@@ -132,23 +132,32 @@ class Dropdown extends React.Component {
     const disabledClass = mapToCssModules('disabled', this.props.cssModule);
 
     const items = container.querySelectorAll(`.${menuClass} .${itemClass}:not(.${disabledClass})`);
+    const char = String.fromCharCode(e.which).toLowerCase();
+    const itemsOfChar = char !== '' &&
```

Comment:
`Array.from` is is not supported in IE and also does not get transpiled. It would require a polyfill.